Use as library: Can't load "default.properties"

When using JBake as library in an isolated classloader, an exception is thrown, indicating that the "default.propeties" files can't be loaded. Althoght a file "default.properties" is located inside the `jbake-core.jar" as expected.

The setup is as follows:
* The JBake dist ZIP 2.6.4 is extracted into a `jbake-dist` dir
* An  `java.net.URLClassLoader`is created with all JAR from `jbake-dist/lib/` dir, but no parent classloader
* The `org.jbake.launcher.Main` class is loaded and the `main` method invoked
* The following exception is thrown:

```
Caused by: org.apache.commons.configuration.ConfigurationException: Cannot locate configuration source default.properties
        at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:259)
        at org.apache.commons.configuration.AbstractFileConfiguration.load(AbstractFileConfiguration.java:238)
        at org.apache.commons.configuration.AbstractFileConfiguration.<init>(AbstractFileConfiguration.java:158)
        at org.apache.commons.configuration.PropertiesConfiguration.<init>(PropertiesConfiguration.java:252)
        at org.jbake.app.configuration.ConfigUtil.load(ConfigUtil.java:47)
        at org.jbake.app.configuration.ConfigUtil.loadConfig(ConfigUtil.java:58)
        at org.jbake.app.configuration.JBakeConfigurationFactory.createDefaultJbakeConfiguration(JBakeConfigurationFactory.java:29)
        at org.jbake.launcher.Main.run(Main.java:78)
```

## Details

A sketch of the code (Sala):
```scala
val cl = new URLClassLoader(classpath, null)
val mainClass = cl.loadClass("org.jbake.launcher.Main")
val mainMethod = mainClass.getMethod("main", Seq(classOf[Array[String]]): _*)
mainMethod.invoke(null, args)
```

The JARs in the classloader:
```
Creating Classloader with classpath: [Vector(/tmp/jbake-2.6.4-bin/lib/args4j-2.33.jar,
 /tmp/jbake-2.6.4-bin/lib/asciidoctorj-1.5.8.1.jar,
 /tmp/jbake-2.6.4-bin/lib/asciidoctorj-diagram-1.5.11.jar,
 /tmp/jbake-2.6.4-bin/lib/asm-5.0.3.jar,
 /tmp/jbake-2.6.4-bin/lib/asm-analysis-5.0.3.jar,
 /tmp/jbake-2.6.4-bin/lib/asm-tree-5.0.3.jar,
 /tmp/jbake-2.6.4-bin/lib/asm-util-5.0.3.jar,
 /tmp/jbake-2.6.4-bin/lib/attoparser-2.0.5.RELEASE.jar,
 /tmp/jbake-2.6.4-bin/lib/autolink-0.6.0.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-collections-3.2.2.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-configuration-1.10.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-io-2.6.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-jexl-2.1.1.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-lang-2.6.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-lang3-3.8.1.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-logging-1.2.jar,
 /tmp/jbake-2.6.4-bin/lib/commons-vfs2-2.2.jar,
 /tmp/jbake-2.6.4-bin/lib/concurrentlinkedhashmap-lru-1.4.2.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-abbreviation-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-anchorlink-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-aside-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-autolink-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-definition-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-emoji-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-escaped-character-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-footnotes-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-gfm-strikethrough-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-gfm-tasklist-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-ins-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-jekyll-front-matter-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-superscript-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-tables-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-toc-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-typographic-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-wikilink-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-ext-yaml-front-matter-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-formatter-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-jira-converter-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-profile-pegdown-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/flexmark-util-0.40.8.jar,
 /tmp/jbake-2.6.4-bin/lib/freemarker-2.3.28.jar,
 /tmp/jbake-2.6.4-bin/lib/groovy-2.5.5.jar,
 /tmp/jbake-2.6.4-bin/lib/groovy-dateutil-2.5.5.jar,
 /tmp/jbake-2.6.4-bin/lib/groovy-templates-2.5.5.jar,
 /tmp/jbake-2.6.4-bin/lib/groovy-xml-2.5.5.jar,
 /tmp/jbake-2.6.4-bin/lib/jade4j-1.2.7.jar,
 /tmp/jbake-2.6.4-bin/lib/javassist-3.20.0-GA.jar,
 /tmp/jbake-2.6.4-bin/lib/javax.servlet-api-3.1.0.jar,
 /tmp/jbake-2.6.4-bin/lib/jbake-core-2.6.4.jar,
 /tmp/jbake-2.6.4-bin/lib/jbake-dist-2.6.4.jar,
 /tmp/jbake-2.6.4-bin/lib/jcommander-1.35.jar,
 /tmp/jbake-2.6.4-bin/lib/jetty-http-9.2.26.v20180806.jar,
 /tmp/jbake-2.6.4-bin/lib/jetty-io-9.2.26.v20180806.jar,
 /tmp/jbake-2.6.4-bin/lib/jetty-server-9.2.26.v20180806.jar,
 /tmp/jbake-2.6.4-bin/lib/jetty-util-9.2.26.v20180806.jar,
 /tmp/jbake-2.6.4-bin/lib/jna-4.5.0.jar,
 /tmp/jbake-2.6.4-bin/lib/jna-platform-4.5.0.jar,
 /tmp/jbake-2.6.4-bin/lib/jruby-complete-9.1.17.0.jar,
 /tmp/jbake-2.6.4-bin/lib/json-simple-1.1.1.jar,
 /tmp/jbake-2.6.4-bin/lib/jsoup-1.11.3.jar,
 /tmp/jbake-2.6.4-bin/lib/jul-to-slf4j-1.7.25.jar,
 /tmp/jbake-2.6.4-bin/lib/logback-classic-1.2.3.jar,
 /tmp/jbake-2.6.4-bin/lib/logback-core-1.2.3.jar,
 /tmp/jbake-2.6.4-bin/lib/logging, /tmp/jbake-2.6.4-bin/lib/ognl-3.1.12.jar,
 /tmp/jbake-2.6.4-bin/lib/orientdb-core-2.2.37.jar,
 /tmp/jbake-2.6.4-bin/lib/parboiled-core-1.1.7.jar,
 /tmp/jbake-2.6.4-bin/lib/parboiled-java-1.1.7.jar,
 /tmp/jbake-2.6.4-bin/lib/pegdown-1.6.0.jar,
 /tmp/jbake-2.6.4-bin/lib/slf4j-api-1.7.25.jar,
 /tmp/jbake-2.6.4-bin/lib/snappy-java-1.1.0.1.jar,
 /tmp/jbake-2.6.4-bin/lib/thymeleaf-3.0.11.RELEASE.jar,
 /tmp/jbake-2.6.4-bin/lib/unbescape-1.1.6.RELEASE.jar)]
```

The JVM:
```
$ java -version
openjdk version "1.8.0_191"
OpenJDK Runtime Environment (IcedTea 3.10.0) (Gentoo icedtea-3.10.0)
OpenJDK 64-Bit Server VM (build 25.191-b12, mixed mode)
```

--- a/jbake-core/src/main/java/org/jbake/app/configuration/ConfigUtil.java
+++ b/jbake-core/src/main/java/org/jbake/app/configuration/ConfigUtil.java
@@ -9,6 +9,7 @@ import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.File;
+import java.net.URL;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
@@ -44,7 +45,8 @@ public class ConfigUtil {
         if (customConfigFile.exists()) {
             config.addConfiguration(new PropertiesConfiguration(customConfigFile));
         }
-        config.addConfiguration(new PropertiesConfiguration(DEFAULT_CONFIG_FILE));
+        URL defaultPropertiesLocation = this.getClass().getClassLoader().getResource(DEFAULT_CONFIG_FILE);
+        config.addConfiguration(new PropertiesConfiguration(defaultPropertiesLocation));
         config.addConfiguration(new SystemConfiguration());
         return config;
     }

